modules.json

```diff
@@ -497,5 +497,17 @@
             "list": "logevents",
             "lelimit": "3"
         }
+    },
```

Comment:
We don't have to add the params in this file as the module requires more than one `GET` request. Just realized that I ignored it here https://github.com/wikimedia/mediawiki-api-demos/pull/128/files. Could you remove these params and also the one related to watchlist so that I can merge this pull request?